### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by karenav**

### CODE REVIEW

**Review by tbarnahor — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
-This is the repo of Karen Avner and Tal Bar Nahor's capstone project!
-The name of our project is "food-mood", 
-and it's a web application that helps you decide where to order food from!
+# Heading 1
```
Comment (szwbar) [inline README.md]: I think the heading should have some content :)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
+Karen Avner and Tal Bar Nahor's Google STEP internship project!
+Our project's name is "food-mood", and it's a web application that helps you decide where to order food from.
```
Comment (szwbar) [inline README.md]: nit: Food-Mood?

**Review by szwbar — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (karenav) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (karenav) [reply README.md]: hahaha ok that was a funny misunderstanding on my side. fixed

**Pull request merged**